### ISSUE TITLE
How to apply layout with mapping to edge attribute?

### ISSUE THREAD

**Issue by twesleyb** [cytoscape/RCy3#59]
The `RCy3` package is excellent, thank you!

I'd like to apply the __perfuse force-directed__ layout with mapping to edge weight using RCy3. 
 Is this possible?

I can apply the force-directed layout to my network with:
```
layoutNetwork(layout.name = "force-directed")
```
However, I can't figure out how to use my edge attribute `weight` in this layout.
The cytoscape option '_Edge column that contains the weights:_' is not listed with `getLayoutPropertyNames()`, so maybe this isn't possible?

```
getLayoutPropertyNames(layout.name = "force-directed")
```
[1] "numIterations"            "defaultSpringCoefficient" "defaultSpringLength"     
[4] "defaultNodeMass"          "isDeterministic"          "singlePartition"

Hopefully, I have not missed an obvious answer in the documentation, my apologies if I have.

**Comment by AlexanderPico** [comment]
Sorry for the delayed response!  You found a bug in the valued returned by `getLayoutPropertyNames`. I'll get to fixing that soon. But, fortunately, in the meantime, you can do what and just ignore the truncated list of property names!

In general, you can always go to the source for the documentation on `Commands` based functions like this one. There are two ways to do this:

1. In Cytoscape, View > Show Automation Panel. Then type `help layout force-directed' and you'll see all the properties (including the ones incorrectly truncated from the list you're getting.
2. In RCy3, run `commandsAPI()` and you'll get the interactive Swagger version of the same docs.

You'll see that `edgeAttribute` is indeed available as a parameter that takes the name of an edge column with numeric values.

**Comment by AlexanderPico** [comment]
Dev Note:  implementation depended on CyREST, which is source of bug:
https://github.com/cytoscape/cyREST/issues/100

In the meantime, we can reimplement to use `commandsHelp` instead.

**Comment by AlexanderPico** [comment]
Fixed in version 2.4.1

**Comment by twesleyb** [comment]
Thank you Alexander!
